directanswercards/documentsearch-standard/component.js

```diff
@@ -21,11 +21,16 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
       snippetValue = Formatter.highlightField(snippet.value, snippet.matchedSubstrings);
     }
 
+    let viewDetailsUrl = relatedItemData.website || (relatedItemData.fieldValues && relatedItemData.fieldValues.landingPageUrl);
+    if (HitchhikerJS.useragent.isChrome()) {
```

Comment:
don't have any context but why chrome only?

Reply:
this is a chrome feature thing, doesn't work on other browsers

Reply:
ohhh my bad I honestly forgot what this item was lol